gallery/gallery.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image Gallery</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+    <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="./gallery-styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="nav-container">
+        <nav>
+          <div class="menu-toggle" id="mobile-menu">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+          <div class="nav-menu">
+            <a class="nav-link" href="../index.html">Home</a>
+            <a class="nav-link" href="../resume/resume.html">Resume</a>
+            <a class="nav-link" href="../about/about.html">About Me</a>
+            <a class="nav-link" href="../gallery/gallery.html">My Work</a>
+            <a class="nav-link" href="https://medium.com/@stenefra">My Blog</a>
+            <a class="nav-link" href="../getintouch/contact.html">Contact Me</a>
+          </div>
+        </nav>
+      </div>
+      <img src="../photos/new-aca-logo.png" class="logo" alt="austin coding academy logo" />
+    </header>
+    <main>
+      <div class="row">
+        <div class="column">
+          <div class="container">
+            <img class="image" src="https://source.unsplash.com/random?sig=1" alt="" />
```

Comment:
`alt="random image from unsplash.com"` or something. always good to have that filled out.